spec/features/bucket_lists/user_creates_bucket_list_spec.rb

```diff
@@ -33,21 +33,21 @@
     fill_in 'Title', with: 'Europe'
     click_button 'Create Your Bucket List!'
 
-    expect(page).to have_content("You need to sign in or sign up before continuing.")
+    expect(page).to have_content("You can do that after you sign in or sign up!")
     expect(page).to have_content("Email")
     expect(page).to have_content("Password")
   end
 
+  scenario "authenticated user fails to sucessfully create a bucket list" do
+    visit new_user_session_path
+    sign_in
+    visit root_path
+    click_link 'New Bucket List'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.